src/CommunityToolkit.Maui.Core/Handlers/Popup/PopUpHandler.android.cs

```diff
@@ -91,9 +91,6 @@ public static void MapColor(PopupHandler handler, IPopup view)
 	public static void MapSize(PopupHandler handler, IPopup view)
 	{
 		ArgumentNullException.ThrowIfNull(handler.Container);
-
```

Comment:
However, since the MapSize method was being called when the VerticalOptions and HorizontalOptions properties were changed, it is necessary to correct it as shown below.

[src\CommunityToolkit.Maui\HandlerImplementation\Popup\Popup.android.cs]

    void OnPropertyChanged(object? sender, PropertyChangedEventArgs e)
    {
        if (e.PropertyName == nameof(Size) ||
            e.PropertyName == nameof(HorizontalOptions) ||
            e.PropertyName == nameof(VerticalOptions))
        {
            if (Handler?.MauiContext is IMauiContext mauiContext)
            {
                var platformPopup = this.ToHandler(mauiContext);
                if (platformPopup.PlatformView is Dialog dialog &&
                    platformPopup.VirtualView is IPopup pPopup &&
                    pPopup.Content?.ToPlatform(mauiContext) is AView container)
                {
                    SetSize(dialog, this, pPopup, container);
                    CommunityToolkit.Maui.Core.Views.PopupExtensions.SetAnchor(dialog, pPopup);
                }
            }
        }
    }

[src\CommunityToolkit.Maui\HandlerImplementation\Popup\Popup.windows.cs]

    void OnPropertyChanged(object? sender, PropertyChangedEventArgs e)
    {
        if (e.PropertyName == nameof(Size) ||
            e.PropertyName == nameof(HorizontalOptions) ||
            e.PropertyName == nameof(VerticalOptions))
        {
            if (Handler?.MauiContext is IMauiContext mauiContext)
            {
                var platformPopup = this.ToHandler(mauiContext);
                if (platformPopup.PlatformView is Microsoft.UI.Xaml.Controls.Primitives.Popup dialog &&
                    platformPopup.VirtualView is IPopup pPopup &&
                    pPopup.Content?.ToPlatform(mauiContext) is Microsoft.UI.Xaml.FrameworkElement container)
                {
                    SetSize(dialog, this, pPopup, mauiContext, container);
                    CommunityToolkit.Maui.Core.Views.PopupExtensions.SetLayout(dialog, pPopup, mauiContext);
                }
            }
        }
    }

[src\CommunityToolkit.Maui.Core\Handlers\Popup\PopupHandler.shared.cs]

    public static IPropertyMapper<IPopup, PopupHandler> PopUpMapper = new PropertyMapper<IPopup, PopupHandler>(ElementMapper)
    {
        [nameof(IPopup.Anchor)] = MapAnchor,
        [nameof(IPopup.Color)] = MapColor,
    #if IOS || MACCATALYST || TIZEN
        [nameof(IPopup.Size)] = MapSize,
        [nameof(IPopup.VerticalOptions)] = MapSize,
        [nameof(IPopup.HorizontalOptions)] = MapSize,
    #endif
        [nameof(IPopup.CanBeDismissedByTappingOutsideOfPopup)] = MapCanBeDismissedByTappingOutsideOfPopup
    };

Therefore, the MapSize method is not required on Windows and Android.
The following must be removed.

[src\CommunityToolkit.Maui.Core\Handlers\Popup\PopUpHandler.android.cs]

    public static void MapSize(PopupHandler handler, IPopup view)
    {
        ArgumentNullException.ThrowIfNull(handler.Container);
    }

[src\CommunityToolkit.Maui.Core\Handlers\Popup\PopUpHandler.windows.cs]

    public static void MapSize(PopupHandler handler, IPopup view)
    {

    }

I will update the fix after verifying operation.